qiskit/algorithms/time_evolvers/classical_methods/evolve.py

```diff
@@ -93,6 +93,7 @@ def _evaluate_aux_ops(
         Tuple of the mean and standard deviation of the aux operators for a given state.
     """
     op_means = np.array([np.real(state.conjugate().dot(op.dot(state))) for op in aux_ops])
+    # TODO: returns only means? what about std?
```

Comment:
The function do not match neither description nor type annotation

Reply:
Yep, that's also inconsistent with other time evolution methods. For this PR, could you update the doc to reflect that it only returns the means? I'll open an issue to track this separately.